windows/bleachbit_lang.nsh

```diff
@@ -0,0 +1,1810 @@
+!endif
+
+!ifdef LANG_GEORGIAN
+	LangString BLEACHBIT_UPGRADE_UNINSTALL ${LANG_GEORGIAN} "${prodname} is already installed. Click 'OK' to uninstall the old version before upgrading, or click 'Cancel' to abort the upgrade."
+	LangString BLEACHBIT_COMPONENT_CORE_TITLE ${LANG_GEORGIAN} "Core"
+	LangString BLEACHBIT_COMPONENT_CORE_DESCRIPTION ${LANG_GEORGIAN} "Installs the BleachBit Core"
+	LangString BLEACHBIT_COMPONENTGROUP_SHORTCUTS_TITLE ${LANG_GEORGIAN} "Shortcuts"
+	LangString BLEACHBIT_COMPONENTGROUP_SHORTCUTS_DESCRIPTION ${LANG_GEORGIAN} "Creates Shortcuts for BleachBit"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_TITLE ${LANG_GEORGIAN} "Start menu"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_DESCRIPTION ${LANG_GEORGIAN} "Creates Start menu Shortcuts for BleachBit"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_LINK_NO_UAC ${LANG_GEORGIAN} "No User Account Control"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_LINK_DEBUGGING_TERMINAL ${LANG_GEORGIAN} "Debugging Terminal"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_LINK_HOME_PAGE ${LANG_GEORGIAN} "Home Page"
+	LangString BLEACHBIT_COMPONENT_STARTMENU_LINK_UNINSTALL ${LANG_GEORGIAN} "Uninstall"
+	LangString BLEACHBIT_COMPONENT_DESKTOP_TITLE ${LANG_GEORGIAN} "Desktop"
+	LangString BLEACHBIT_COMPONENT_DESKTOP_DESCRIPTION ${LANG_GEORGIAN} "Creates Desktop Shortcut for BleachBit"
+	LangString BLEACHBIT_COMPONENT_QUICKLAUNCH_TITLE ${LANG_GEORGIAN} "Quick launch"
+	LangString BLEACHBIT_COMPONENT_QUICKLAUNCH_DESCRIPTION ${LANG_GEORGIAN} "Creates Quick launch Shortcut for BleachBit"
+	LangString BLEACHBIT_COMPONENT_AUTOSTART_TITLE ${LANG_GEORGIAN} "Autostart"
+	LangString BLEACHBIT_COMPONENT_AUTOSTART_DESCRIPTION ${LANG_GEORGIAN} "Creates Autostart Shortcut for BleachBit"
+	LangString BLEACHBIT_COMPONENT_TRANSLATIONS_TITLE ${LANG_GEORGIAN} "Translations"
+	LangString BLEACHBIT_COMPONENT_TRANSLATIONS_DESCRIPTION ${LANG_GEORGIAN} "BleachBit Translations"
+	LangString BLEACHBIT_COMPONENT_INTEGRATESHRED_TITLE ${LANG_GEORGIAN} "Integrate Shred"
+	LangString BLEACHBIT_COMPONENT_INTEGRATESHRED_DESCRIPTION ${LANG_GEORGIAN} "Integrate BleachBit's Shred into Windows Explorer"
+	LangString BLEACHBIT_SHELL_TITLE ${LANG_GEORGIAN} "Shred with BleachBit"
+	LangString BLEACHBIT_MUI_FINISHPAGE_LINK ${LANG_GEORGIAN} "Visit the ${prodname} web site"
+	LangString BLEACHBIT_UNINSTALLTEXT ${LANG_GEORGIAN} "BleachBit will be uninstalled from the following folder. Click 'Uninstall' to start the uninstallation. WARNING: The uninstaller completely removes the installation directory including any files (such as custom cleaners) that you may have added or changed!"
+!endif
+
+!ifdef LANG_GERMAN
+	; Translation 2019-04-01 - 2019-04-02 by https://github.com/Tobias-B-Besemer
```

Comment:
It seems this whole file should be generated by a script from the `.po` files downloaded from Launchpad. Managing the translation another way could cost much effort, lead to copying errors, and cause delays.

Here's the program I use to do something like this
https://github.com/bleachbit/bleachbit-misc/blob/master/extract_desktop.py

Reply:
If you want my help with this, let me know. I might be able to make fast progress if I had a table or map that translates the ISO codes used for .po files to NSIS Language names. For example, de=German, es=Spanish

Anyway, the the goal would be to let translators work in Launchpad, and then the script makes it easy to get the translations from Launchpad into this NSIS file

Reply:
This should IMHO be done in a new PR!
I opened a issue for it (#529) and wrote something to it...
As long will BB only have complete support for German... But this is IMHO not so terrible as long we get the from me generated PO files into Launchpad. Because the translators need time to translate the strings...

Reply:
Addition: In my latest release I have done a lot about language! Also set the ISO codes used for .po files in relation to NSIS Language names... But later more to it!